test/attr/attr_final.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -swift-version 4
```

Comment:
This is not a change we should make just yet. We're using our existing test suite as source compatibility tests.